packages/legend-query/src/stores/QueryBuilderState.ts

```diff
@@ -118,6 +119,7 @@ export class QueryBuilderState {
   resultSetModifierState: QueryResultSetModifierState;
   resultState: QueryBuilderResultState;
   queryTextEditorState: QueryTextEditorState;
+  getAllFunctionState?: SimpleFunctionExpression | undefined;
```

Comment:
I don't think this state aligns with our UI state. I would add to the `querySetupState` state. Maybe add a property `versionPropertyParameter?: ValueSpecification` which should hold the value of that parameter (either a variable like `$businessDate` or an instance of date like `11-10-12`)